### ISSUE TITLE
Themes

### ISSUE THREAD

**Issue by cranndarach** [cranndarach/color-comm#4]
Some pre-designed themes to select.
- [x] Probably cookies to save choice so that user does not have to set the theme on every page. 
- [x] Include variety of themes (encourage contribution?) to make up for not allowing full customization
- [ ] _Possibly_ a script to populate a "themes" section rather than copy-paste into every page
- [x] Still definitely allow for customizing font size and face, at minimum
- [ ] Probably option to turn off any "on hover" style

**Comment by cranndarach** [comment]
[ Moved checklist to first post ]
